client/src/components/ClueModal.js

```diff
@@ -53,9 +56,20 @@ const ClueModal = (props) => {
         setNumOfGuesses(e.target.value);
     }
 
+    const isOneAlphabeticWord = (word) => {
```

Comment:
Checking if the word is not one of the clue words

Reply:
This is implemented.